### ISSUE TITLE
Revert LibreOffice UI fix

### ISSUE THREAD

**Issue by stelariusinfinitek** [tista500/Adapta#190]
**Reporter info**

 ```
 * Distribution - Arch Linux
 * Gtk+ 3.0 version - 3.20.8
 * Desktop environment - Xfce4
 ```

**Adapta version (if you knew)** 3.21.4.105

**Related Application and/or Shell theming (or name of widget)** LibreOffice

**Actual issue** You applied a fix for the UI issue with LibreOffice. That issue is now fixed in LibreOffice 5.2.0 and I think you can revert the fix (or remove it, I hope the term is okay). 

**Steps to reproduce (if you knew)** Open LibreOffice Writer. A dark bar appears at the top.

![screenshot_2016-08-05_00-11-40](https://cloud.githubusercontent.com/assets/18717816/17418899/7f140342-5aa3-11e6-9260-be7aa5d16b88.png)

**Expected behaviour**

**Other Note (feature-request, question, etc...)**
Before reporting this, I had an update for gtk3 so I reinstalled the theme, just to be sure.

**Comment by stelariusinfinitek** [comment]
I'm so sorry, I don't understand, it was supposed to be fixed. I switched to Arc-Darker just for testing and this is what I saw : 
  
![2016-08-05-003253_1366x768_scrot](https://cloud.githubusercontent.com/assets/18717816/17419042/402052c0-5aa4-11e6-8cf2-d84aecdcbe5e.png)

I will do more testing and report back. I should have done the tests before, I'm sorry. Even if it's not fixed, can you change the color of that black bar to fit to the rest of the UI?

**Comment by tista500** [comment]
Ah, seems to be fixed (if you mean the appearance of MenuBar)...

But.

I don't trust their release note at all, so please wait until 5.2.0 has landed on our main repository, I need to verify it by myself on my local machines... Anyway reverting is quite easy.

Regards.

**Comment by tista500** [comment]
I now checked pre-released 5.2.0 and 5.2.0 seems to ignore my fixes already and renders MenuBar and MenuItems properly with correct CSS-based theming. I don't think I need to revert my fixes in case integrating 5.2.0 specifically...

So I will keep my fixes until 5.2.0 has landed on most of major distributions, and after that, I will drop my fixes when 5.2.0 became the stable-version of them.

Regards.
